pkg/util/scrubber/default.go

```diff
@@ -94,11 +94,17 @@ func AddDefaultReplacers(scrubber *Scrubber) {
 		Regex: regexp.MustCompile(`(?i)([a-z][a-z0-9+-.]+://|\b)([^:]+):([^\s|"]+)@`),
 		Repl:  []byte(`$1$2:********@`),
 	}
-	passwordReplacer := matchYAMLKeyPart(
+	yamlPasswordReplacer := matchYAMLKeyPart(
 		`(pass(word)?|pwd)`,
 		[]string{"pass", "pwd"},
 		[]byte(`$1 "********"`),
 	)
+	// replace all alphanum + special chars except quotes
+	passwordReplacer := Replacer{
+		// see test cases for match patterns
+		Regex: regexp.MustCompile(`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)( = |=|: \"?)([0-9A-Za-z#!$%&()*+,\-./:<=>?@[\\\]^_{|}~]+)`),
```

Comment:
What about `;` ? I know that could scrub too much (eg. in the SQL connection error), but better too much than not enough

Reply:
Left as is to avoid scrubbing too much in SQL queries or URLs